ingestion/src/metadata/cmd.py

```diff
@@ -123,4 +131,80 @@ def report(config: str) -> None:
         ) from exc
 
 
+@metadata.command()
+@click.option(
+    "-t",
+    "--type",
+    help="Workflow config",
+    required=True,
+)
+@click.option(
+    "-p",
+    "--path",
+    help="Workflow config",
+    required=False,
+)
+def docker_run(type: str, path: str = "") -> None:
+    """
+    Checks Docker Memory Allocation
+    Run Local Docker - metadata docker-run -t local
+    Run Latest Release Docker - metadata docker-run -t latest
+    """
+    try:
+        client = docker.from_env()
+        docker_info = client.info()
+        calc_gb = 1024 * 1024 * 1000
+        if docker_info["MemTotal"] < (3 * 1024 * 1024 * 1000):
+            raise MemoryError(
+                f"Please Allocate More memory to Docker.\nRecommended: 4GB\nCurrent: "
+                f"{round(float(docker_info['MemTotal']) / calc_gb, 2)}"
+            )
+        if type == "local":
+            logger.info("Running Local Docker")
+            if path == "":
+                raise ValueError("Please Provide Path to local docker-compose.yml file")
+            else:
+                subprocess.run(
+                    f"docker compose -f {path} up --build -d",
+                    shell=True,
+                )
+        elif type == "latest":
+            logger.info("Running Latest Release Docker")
+            r = requests.get(
+                "https://raw.githubusercontent.com/open-metadata/OpenMetadata/main/docker/metadata/docker-compose.yml"
+            )
+            open("/tmp/docker-compose.yml", "wb").write(r.content)
+            subprocess.run(f"docker compose -f /tmp/docker-compose.yml up", shell=True)
+        else:
+            raise ValueError("Please Enter one of the following. 'local' or 'latest'")
+    except Exception as err:
+        logger.error(err)
+
+
+@metadata.command()
+def docker_stop() -> None:
+    """
+    Stops all the running containers
+    metadata docker-stop
+    """
+    client = docker.from_env()
+    running_containers = client.containers.list()
+    logger.info("Stopping Running Containers")
+    for container in running_containers:
```

Comment:
same goes here , where we are operating on containers make sure those are related to openmetadata. This will list all the running_containers

Reply:
as we keep our docker-file from release under /tmp, the label will be `tmp`, I have added support to both.
if it's tmp or local-metadata, it will stop those running containers likewise